app/src/main/java/org/android/go/sopt/data/datasource/local/GSDataStore.kt

```diff
@@ -34,6 +38,13 @@ class GSDataStore(context: Context) {
         set(value) = dataStore.edit { putString(USER_FAVORITE_SONG, value) }
         get() = dataStore.getString(USER_FAVORITE_SONG, "") ?: ""
 
+    fun clear() {
+        dataStore.edit {
+            clear()
+            commit()
+        }
+    }
```

Comment:
```suggestion
    fun clear() {
        dataStore.edit { clear() }
    }
```
ktx 쓰면 commit() 생략해두 되지 않나요?